src/panoptes/pocs/focuser/serial.py

```diff
@@ -18,90 +17,63 @@ def __init__(self, *args, **kwargs):
         """Initialize an AbstractSerialMount for the port defined in the config.
             Opens a connection to the serial device, if it is valid.
         """
-
         super().__init__(*args, **kwargs)
 
         # Check that this node hasn't already been assigned to another focuser device
         if self.port in AbstractSerialFocuser._assigned_nodes:
-            message = 'Device node {} already in use!'.format(self.port)
-            self.logger.error(message)
-            warn(message)
+            self.logger.error(f"Device node {self.port} already in use!")
             return
 
         try:
             self.connect(self.port)
-        except (serial.SerialException,
-                serial.SerialTimeoutException,
-                AssertionError) as err:
-            message = 'Error connecting to {} on {}: {}'.format(self.name, self.port, err)
-            self.logger.error(message)
-            warn(message)
+        except Exception as err:
+            self.logger.error(f"Error connecting to {self.name} on {self.port}: {err!r}")
             return
 
         AbstractSerialFocuser._assigned_nodes.append(self.port)
         self._is_moving = False
         self._initialise()
 
+        # Move to the initial position
+        initial_position = kwargs.get("initial_position", None)
+        if initial_position is not None:
+            self.logger.debug(f"Initial position for {self}: {initial_position}")
+            self.move_to(initial_position)
```

Comment:
We have a `initial_position` and (after #1134) `zero_position` that could potentially be consolidated.